cmake/thirdparty/SetupSeracThirdParty.cmake

```diff
@@ -136,6 +136,49 @@ if (NOT SERAC_THIRD_PARTY_LIBRARIES_FOUND)
     endif()
     message(STATUS "Sundials support is ${SERAC_USE_SUNDIALS}")
 
+    #------------------------------------------------------------------------------
+    # PETSc
+    #------------------------------------------------------------------------------
+    if (PETSC_DIR)
+        serac_assert_is_directory(DIR_VARIABLE PETSC_DIR)
+        set(SERAC_USE_PETSC ON CACHE BOOL "")
```

Comment:
We should have a comment explaining the anti-pattern here. Just something simple that says we are using PETSC/SLEPC through MFEM.